test/components/DayPickerRangeController_spec.jsx

```diff
@@ -12,7 +12,7 @@ import isInclusivelyAfterDay from '../../src/utils/isInclusivelyAfterDay';
 
 import { START_DATE, END_DATE } from '../../constants';
 
-const today = moment().startOf('day');
+const today = moment().startOf('day').hours(12);
```

Comment:
This makes sense. Maybe add a comment about how this is due to the fact that all moment objects passed to interaction handlers from `CalendarDay` are set to noon. Not necessary but could be nice.